yt/yt/client/queue_client/config.cpp

```diff
@@ -67,15 +71,30 @@ void TQueueStaticExportConfig::Register(TRegistrar registrar)
         .Default(false);
 
     registrar.Postprocessor([] (TThis* config) {
-        if (config->ExportPeriod.GetValue() % TDuration::Seconds(1).GetValue() != 0) {
-            THROW_ERROR_EXCEPTION("The value of \"export_period\" must be a multiple of 1000 (1 second)");
+        if (config->ExportPeriod) {
+            if (config->ExportPeriod->GetValue() % TDuration::Seconds(1).GetValue() != 0) {
+                THROW_ERROR_EXCEPTION("The value of \"export_period\" must be a multiple of 1000 (1 second)");
+            }
+            config->ExportSchedule = *config->ExportPeriod;
+        } else if (config->ExportCronExpression) {
+            try {
+                config->ExportSchedule.emplace<TCronExpression>(*config->ExportCronExpression);
+            } catch (const yexception& exc) {
+                THROW_ERROR_EXCEPTION(
+                    "The value of \"export_cron\" is not a well-formed CRON expression (Value: '%v', Error: '%v')",
```

Comment:
Unlike logs, in exceptions we format everything within the exception text (sometimes with Qv for automatic quotation marks around strings), and extraneous data via TErrorAttribute (you can codesearch usages). 

So here it would be something like `Export cron expression %Qv is not well-formed`. 
And the error itself can be added as a child-error, via the `<<` operator (same as attributes). Should work for yexception just as well as for ours.

So, something like
```THROW_ERROR_EXCEPTION("Export cron expression %Qv is not well-formed", *config->ExportCronExpression)
    << exc;```
    
Also, we tend to use `ex` instead of `exc`.